lib/common.sh

```diff
@@ -638,6 +638,20 @@ common::argc_validate () {
   fi
 }
 
+###############################################################################
+# Get the md5 hash of a file
+# @param file - The file
+# @print the md5 hash
+common::md5 () {
+  local file=$1
+
+  if which md5 >/dev/null 2>&1; then
+    md5 -q "$1"
```

Comment:
nit: is there a reason, why we don't use `$file` here?
```suggestion
    md5 -q "$file"
```